protobuf/fmp/fmp.proto

```diff
@@ -18,4 +21,10 @@ message Device {
     string sitecode = 4;
```

Comment:
should this starts from 1?

Reply:
github is just hiding the other lines, all of 1-6 are there.